tests/e2escenarios/e2e_test.go

```diff
@@ -37,6 +39,36 @@ var _ = Describe("E2E Test", func() {
 		}, 120*time.Second, 15*time.Second).Should(Equal(assertString))
 	}
 
+	sendDataEntry := func(url string) map[string]interface{} {
+		values := map[string]interface{}{"name": "joe",
+			"location": "tokyo",
+			"age":      23,
+		}
+		json_data, err := json.Marshal(values)
+		Expect(err).To(BeNil())
+		resp, err := http.Post(fmt.Sprintf("http://%s/api/newuser", url), "application/json", bytes.NewBuffer(json_data))
+		Expect(err).To(BeNil())
+		var res map[string]interface{}
+		err = json.NewDecoder(resp.Body).Decode(&res)
+		Expect(err).To(BeNil())
+		return res
+	}
+	receiveData := func(url string) string {
+		resp, err := http.Get(fmt.Sprintf("http://%s", url))
+		Expect(err).ToNot(HaveOccurred())
+		defer resp.Body.Close()
+		body, _ := io.ReadAll(resp.Body)
+		return string(body)
+	}
+
+	failcheck := func(url string) string {
+		resp, err := http.Get(fmt.Sprintf("http://%s", url))
+		Expect(err).To(BeNil())
+		defer resp.Body.Close()
+		body, _ := io.ReadAll(resp.Body)
+		return string(body)
```

Comment:
I don't understand the difference between the `failcheck` and `receiveData` functions. They are doing the same thing, no?

Reply:
Thanks, will update this one